packages/ensnode-sdk/src/shared/deserialize.ts

```diff
@@ -93,7 +93,7 @@ export function deserializeBlockRef(
   return parsed.data;
 }
 
-export function deserializeDuration(maybeDuration: string, valueLabel?: string): Duration {
+export function deserializeDuration(maybeDuration: unknown, valueLabel?: string): Duration {
```

Comment:
Included `makeDurationSchema` call uses `z.coerce`, so we can downcast the `maybeDuration` to `unknown` and make the `deserializeDuration` function more versatile.